codejail/exceptions.py

```diff
@@ -0,0 +1,28 @@
+"""
+Codejail exceptions.
+"""
+
+
+class CodeJailBaseException(Exception):
```

Comment:
A minor point: I would have left out "Base" here: it doesn't add much, and is too Java for my taste :)